src/battle_main.c

```diff
@@ -1917,16 +1917,16 @@ void SpriteCallbackDummy_2(struct Sprite *sprite)
 {
 }
 
-// not used
-UNUSED static void SpriteCB_Unused_8011E28(struct Sprite *sprite)
+// Unused
+static void SpriteCB_8011E28(struct Sprite *sprite)
 {
     sprite->data[3] = 6;
     sprite->data[4] = 1;
-    sprite->callback = SpriteCB_Unused_8011E28_Step;
+    sprite->callback = SpriteCB_8011E28_Step;
```

Comment:
Since we're renaming them, this is `SpriteCB_InitFlicker` and `SpriteCB_Flicker` from pokeemerald